generators/root/templates/deployment/kubernetes/environments/staging/README.md

```diff
@@ -1,13 +1,39 @@
+# Terraform
+
+Terraform is a tool that allows to automate the creation of servers. The terraform configuration of this repository
+is split into several parts, following the required environments:
+ - `staging`: the configuration for the staging environment
+ - any other environment you need (for instance `production` or `cdn`)
+
+You can find the tutorial to install terraform [here](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
 # Usage
 
 ## Configuration
 
-First you need to create a Scaleway API key and secret [here](https://console.scaleway.com/project/credentials).
+You need first to configure the credentials for the environment you want to use.
 
-Then you need to run initialize terraform with:
+First you need to create an API key and secret [here](https://console.scaleway.com/project/credentials).
 
-```
-terraform init --backend-config="access_key={your scw_access_key}" --backend-config="secret_key={your scw_secret_key}"
-```
+You must then copy the `scaleway project id` in  `terraform/{environment}/main.tf`, as well as the bucket id that you
+can create here [here](https://console.scaleway.com/object-storage/buckets)
```

Comment:
I would recommend to reuse an existing bucket for terraform states to keep the states of all projects in only on bucket to prevent polluting the scaleway account with one bucket per project.